cardano_clusterlib/clusterlib.py

```diff
@@ -1692,6 +1692,46 @@ def get_kes_period(self) -> int:
         """Return last block KES period."""
         return int(self.get_slot_no() // self.slots_per_kes_period)
 
+    def get_kes_period_info(
+        self, issue_op_cert: FileType, operational_certificate_issue_counter: FileType
+    ) -> Dict:
+        """Check an operational certificate and certificate issue counter file.
+
+        Args:
+            issue_op_cert: Operational certificate.
+            operational_certificate_issue_counter: Operational certificate counter.
+
+        Returns:
+            Dict: A dictionary containing all the information.
+        """
+        command_output = self.cli(
```

Comment:
There's a `query_cli` method available that saves us from repeating some of the query arguments and handles query command output processing.